pi/hooks/pre_build

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker run --rm --privileged multiarch/qemu-user-static:register --reset
```

Comment:
Where does this script run?

Reply:
Within the Docker Hub's build container (build environment).

Reply:
Cool, I just found about hooks [here](https://docs.docker.com/docker-hub/builds/advanced/#custom-build-phase-hooks)